packages/reputation-miner/ReputationMinerClient.js

```diff
@@ -464,7 +464,7 @@ class ReputationMinerClient {
         const oppSubmission = await repCycle.getReputationHashSubmission(oppEntry.firstSubmitter);
 
         if (oppSubmission.proposedNewRootHash === ethers.constants.AddressZero){
-          const responsePossible = await repCycle.getResponsePossible(disputeStages.INVALIDATE_HASH, entry.lastResponseTimestamp);
+          const responsePossible = await repCycle["getResponsePossible(uint8,uint256)"](disputeStages.INVALIDATE_HASH, entry.lastResponseTimestamp);
```

Comment:
If we could upgrade everything at once in one go, then yes, we wouldn't need this transitional approach. But we can't, and so to avoid a critical window where an update to either the contracts or the miners _must_ be deployed, I want a series of updates where only one thing must be upgraded at a time.

Reply:
So the plan is to first upgrade the miner, then upgrade the contracts? That makes sense, the new miner will work with the old interface.

Reply:
Yep. But then to get rid of the old interface, we then upgrade the miners again (so they use the new interface, not the deprecated old one) and then finally upgrade the contracts (to remove the deprecated old interface).